### ISSUE TITLE
CSV or JSON output

### ISSUE THREAD

**Issue by raoulschaffner** [Matty9191/ssl-cert-check#91]
thank you very much for the script. the current output is good for human reading from screen. but could you add output options like csv or json so that your output can be processed by a further script, for instance to produce a nice looking web page for ssl cert overview? :-)

**Comment by HeidiRechek** [comment]
> thank you very much for the script. the current output is good for human reading from screen. but could you add output options like csv or json so that your output can be processed by a further script, for instance to produce a nice looking web page for ssl cert overview? :-)

**Comment by sandykadam** [comment]
Yes I'm also interested in json output, it will be great if we can have this feature implemented 👍

**Comment by Matty9191** [comment]
Thanks @svogal for the patch. Let me review it.

**Comment by raoulschaffner** [comment]
thank you @Matty9191 and @svogal for looking into this.

**Comment by svogal** [comment]
Note that:
- It's not JSON but JSONL (every line is a separate JSON object).
- All values are strings.
- There's no escaping of double quote or new line, so it's probably not bulletproof. There could be edge cases when the object breaks.

But it was easier to do that way and it works for me without problems.
